Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -56,8 +56,11 @@ extension Event {
         /// The instant at which the test started.
         var startInstant: Test.Clock.Instant
 
-        /// The number of issues recorded for the test.
-        var issueCount = 0
```

Comment:
Would it be simpler to have:

```swift
var issueCount = [Issue.Severity: Int]()
```

?

Reply:
I like this better, but it seems to have led to a new compiler crash in CI:

> https://ci.swift.org/job/pr-swift-testing-linux-5.10/1630/console

May need to use less sugar on the subscript assignment or some other workaround